AdminUi/apps/admin_ui/lib/core/widgets/identities_data_table/identities_data_table_source.dart

```diff
@@ -19,8 +19,9 @@ class IdentityDataTableSource extends AsyncDataTableSource {
 
   final Locale locale;
   final bool hideTierColumn;
+  final void Function(String address) navigateTo;
```

Comment:
I'm not happy with the function name. Wtb `navigateToIdentity`? And make the parameter named to make sure it's an address for the caller.

Reply:
Changes made.